lib/widgets/content.dart

```diff
@@ -22,7 +23,24 @@ class MessageContent extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return BlockContentList(nodes: content.nodes);
+    return InheritedMessage(message: message, child: BlockContentList(nodes: content.nodes));
+  }
+}
+
+/// Provides access to [message].
+class InheritedMessage extends InheritedWidget {
```

Comment:
I think this is a case of https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-useless-documentation :slightly_smiling_face: 

Not everything in our tree has docs, and that's fine — unlike the Flutter framework, this isn't an API being provided to thousands of developers to consume, so I don't anticipate we'd ever want to turn on a lint that requires all public members to have docs.